src/templates/tag.js

```diff
@@ -17,10 +17,6 @@ function expandStructs(parentedStruct, tags) {
 }
 
 function getTagDependencies(tagStruct, tags) {
-  if (!tagStruct) {
-    return [];
-  }
-
```

Comment:
No longer called when we cannot find an Invader struct for the current tag page.